mm2src/coins/my_tx_history_v2.rs

```diff
@@ -222,7 +222,8 @@ impl<'a, Addr: Clone + DisplayAddress + Eq + std::hash::Hash, Tx: Transaction> T
 
         TransactionDetails {
             coin: self.coin,
-            tx_hex: self.tx.tx_hex().into(),
+            // It should be safe to unwrap here
+            tx_hex: self.tx.tx_hex().expect("tx_hex shouldn't be None!").into(),
```

Comment:
Currently, it's safe, but later there can be other transaction types that don't have TX hex.
How about returning `Vec::default()` instead?

Reply:
> How about returning Vec::default() instead?

I agree